src/blob.rs

```diff
@@ -21,14 +64,33 @@ struct Header {
 
 impl<K> Blob<K>
 where
-    K: Default,
+    K: Default + Send,
 {
+    /// Creates new blob file or openes existing and truncates it
+    pub fn open_new<P: AsRef<Path> + Send + 'static>(path: P) -> BlobOpenFuture<K, P> {
```

Comment:
Why `path` should be static? Obviously it will be generated at runtime by `Storage`